.changeset/tough-bugs-vanish.md

```diff
@@ -0,0 +1,10 @@
+---
+'@clerk/chrome-extension': minor
+'@clerk/localizations': minor
+'@clerk/clerk-js': minor
+'@clerk/nextjs': minor
+'@clerk/clerk-react': minor
+'@clerk/types': minor
```

Comment:
These are enough. I think there might have been a partial merge that left some additional files while you were running the changeset command 
```suggestion
'@clerk/clerk-js': minor
'@clerk/clerk-react': minor
'@clerk/types': minor
```